.pre-commit-config.yaml

```diff
@@ -5,8 +5,8 @@ repos:
     hooks:
       - id: check-clang-format-version
         name: Check clang-format version
-        entry: ./ci/check-clang-format-version.sh
-        language: script
+        entry: python ./ci/check-clang-format-version.py
```

Comment:
explicitly use Python3?

Reply:
Sure. `os.run` is not supported pre python 3.6 anyway.